recipes/mono-msbuild/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class MonoConan(ConanFile):
+    name = "mono-msbuild"
+    description = "The Microsoft Build Engine (MSBuild) is the build platform for .NET and Visual Studio."
+    url = "https://github.com/mono/mono"
```

Comment:
url = location of the recipe (=this repo)
homepage = location of the project (=mono-msbuild repo)
```suggestion
    url = "https://github.com/conan-io/conan-center-index"
```